src/Etl/Dto/EtlResponseDto.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Sherl\Sdk\Etl\Dto;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class EtlResponseDto
```

Comment:
Dans le domaine "ETL" il y a plusieurs endpoints d'API. Il faudrait que le nommage de la classe reflète le nom de l'endpoint qui est utilisé, par exemple : 

```suggestion
class ExtractTransformLoadResponseDto
```

(fichier à renommer également)